app/build.gradle.kts

```diff
@@ -13,9 +13,6 @@ plugins {
 
     // handling secrets.properties
     id("com.google.android.libraries.mapsplatform.secrets-gradle-plugin")
-
-    // google firebase
-    id("com.google.gms.google-services")
 }
```

Comment:
Was there any particular reason for removing the firebase dependencies ?

Reply:
Sorry for the confusion, I wasn't aware of the latest commits in main and messed up the branch. With the latest commit things should be okay.